recipes/wordcloud/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.2.1" %}
+
+package:
+    name: wordcloud
+    version: {{ version }}
+
+source:
+    url: https://github.com/amueller/word_cloud/archive/{{ version }}.tar.gz
+    fn: wordcloud-{{ version }}.tar.gz
+    sha256:
```

Comment:
So are we going to fill this guy in or were we waiting for `conda-build` to start failing? 😜